### ISSUE TITLE
Cache CFFPVerify.cfc in the application scope

### ISSUE THREAD

**Issue by charlesr1971** [cfchef/CFFormProtect-Revamp#1]
@tonyjunkes Hi Tony. Thanks for this amazing alternative to CAPTCHA! 

I'm just wondering, would it be wise to cache

`CFFPVerify.cfc`

Into the **application** scope? Rather than having to instantiate the component each time a form is submitted. 

I am just wondering, if there are any disadvantages to this approach, that you can think of?

**Comment by tonyjunkes** [comment]
Hi @charlesr1971,

Off the top of my head, and glancing over the CFC, I think you’d be fine with caching it with no negative side effects. I want to say I’ve done a similar caching strategy with it in FW/1 once upon a time.

Just know, I haven’t made use of this in ages and never fully tested this conversion from the original library it’s based on, so your mileage may vary.

**Comment by charlesr1971** [comment]
@tonyjunkes Thanks for the advice. Actually, I am using **cffp**, in an FW1 project, at the moment. I think I will add it, as a **service**, so this should get cached automatically? If I remember correctly, **services** are treated as singletons?

**Comment by tonyjunkes** [comment]
@charlesr1971 correct. I dug up an old FW/1 app that used it and while I'm questioning what old me was thinking, I essentially called the CFC from an FW/1 service that ultimately would be treated as a singleton in the rest of the app.
